letsencrypt/cli.py

```diff
@@ -806,12 +806,18 @@ def _restore_required_config_elements(config, renewalparams):
     # int-valued items to add if they're present
     for config_item in INT_CONFIG_ITEMS:
         if config_item in renewalparams and not _set_by_cli(config_item):
-            try:
-                value = int(renewalparams[config_item])
-                setattr(config.namespace, config_item, value)
-            except ValueError:
-                raise errors.Error(
-                    "Expected a numeric value for {0}".format(config_item))
+            config_value = renewalparams[config_item]
+            # the default value for http01_port was None during private beta
+            if config_item == "http01_port" and config_value == "None":
```

Comment:
I did check this with a real renewal config. The value was `"None"` and not `None`. I believe this is similar to what's going on [here](https://github.com/letsencrypt/letsencrypt/pull/2579/files#diff-54b2f468d57ef7436008bd031bc89711L803).